### ISSUE TITLE
No se ajusta bien a pantallas pequeñas

### ISSUE THREAD

**Issue by analca3** [github-users-grx/github-users-grx.github.io#2]
La segunda línea de los créditos no puedo verla en mi pantalla, supongo que habrá que ajustar a los distintos tamaños de ventana. En modo ventana tampoco se ajusta bien la página.

PD: Mola un montón!

**Comment by stringparser** [comment]
¡Gracias! Si, es porque el cuerpo de la página está puesto para que no haga scroll por defecto. Creo que puse más grande el gato y no revisé el pie de página :D. 

PD. Gracias! Abre todos los issues que quieras tio.

**Comment by stringparser** [comment]
Míralo ahora a ver que tal. No tiene una solución fácil. Tengo que buscar una herramienta para hacer testeos de estas cosas :P

**Comment by agarciamontoro** [comment]
Firefox viene ya con una herramienta maravillosa para probar todos los tamaños de pantalla. (Creo que Chrome también)

**Comment by stringparser** [comment]
¿Qué está en la developer tools? No la había visto, voy a curiosear.

**Comment by agarciamontoro** [comment]
Yep. Ahora mismo estoy en el móvil y no te puedo decir qué botoncito era, pero por ahí está. Échale un vistazo porque moooola.

**Comment by stringparser** [comment]
Hay que ver que no le había prestado atención. Y eso que es el primer botón :dancer: 

![image](https://cloud.githubusercontent.com/assets/7457705/7923671/6ccd7754-08b5-11e5-8692-78146cdbe735.png)

**Comment by iblancasa** [comment]
@stringparser no me digas que ves "Hora de Aventuras". Entonces sí que nos vamos a llevar bien jajajajja

**Comment by stringparser** [comment]
@iblancasa iyo, no he visto una cosa más absurdamente buena en la vida. ¡Pues claro que lo veo!

**Comment by analca3** [comment]
Mucho mejor! No está perfecto, pero se nota mucho el cambio! Lástima no tener tiempo de aprender nada de web...

**Comment by stringparser** [comment]
Hahha, es un coñazo. No tiene porqué ser hoy tío hay muchos recursos por ahí aunque de todas formas el aprender a base de golpes es el que mejor funciona :smile:
